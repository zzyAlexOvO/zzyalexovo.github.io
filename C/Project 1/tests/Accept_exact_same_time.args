Wirragulla Cardiff 10:30:23