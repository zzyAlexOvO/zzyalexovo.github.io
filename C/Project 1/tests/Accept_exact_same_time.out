The next train to Cardiff from Wirragulla departs at 10:30:23
